.github/workflows/tests.yml

```diff
@@ -0,0 +1,38 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
```

Comment:
🤔 so you mixed `plone/meta` and `zopefoundation/meta` ?

All cool and good, but I would be interested if you can share your pain points, or reasons on why to do that, or which parts of `plone/meta` (like the github workflows) you don't want/need.

This way, we can add more configuration tweaks to please not only plone core packages, but also addons.

My idea with `plone/meta` has been always to have a common ground for core, but specially also for collective packages, where the quality and tools to double check their status is far more difficult than core 😅 

I'm planning to use it at work where we use gitlab rather than github, so I will add at some point a complete switch to turn on/off github workflows 😄

Reply:
Ah! I actually copied this one from zpretty where it is there since ages.
The reason is that `plone/meta` wanted to run the tests with Plone 6 and I did not find a way to change it :)
So I removed what was generated in the `.github` folder and added something that just worked.